endpoints/services.md

```diff
@@ -68,6 +68,7 @@ Authorization: Bearer $ACCESS_TOKEN
       "service_type": "B",
       "service_type_alias": "Oil and Filters",
       "service_number": 123,
+      "fault_num": 123,
```

Comment:
This should be a string

Reply:
Good catch! These issues should be resolved now